src/extensions/default/BrambleUrlCodeHints/camera/interface.js

```diff
@@ -101,6 +117,98 @@ define(function (require, exports, module) {
         // Update the photo component with the snapped photo
         this.photo.update();
 
+        // pattern from http://codepen.io/SitePoint/full/LVpNjp/
+        // $(function() {
+
+          //var Caman = require('caman').Caman;
+          var canvas = document.getElementById("canvas");  // canvas reference here
```

Comment:
According to http://camanjs.com/api/classes/Caman.html#constructor-instance you should be able to pass a DOM Objet here instead of a selector string, which means you should be able to do something like this:

```js
var canvas = this.canvas;
...
Caman(canvas, img, function() { ... ));
```
Test and confirm I'm right.  You won't need more than one canvas, it's just that you have to initialize it properly.